doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -73,11 +73,13 @@ nRF9160: Asset Tracker v2
 * Added new documentation for Asset Tracker v2 :ref:`asset_tracker_v2_modem_module`.
 * Added support for A-GPS filtered ephemerides.
 * Added new documentation for :ref:`asset_tracker_v2_util_module` and :ref:`api_modules_common`.
+* Updated support for P-GPS preemptive updates and P-GPS coexistence with A-GPS.
 
 nRF9160: Asset Tracker
 ----------------------
 
-* The application is removed.Hence, it is recommended to upgrade to the :ref:`asset_tracker_v2_modem_module` application.
+* The application is removed.
+  Hence, it is recommended to upgrade to the :ref:`asset_tracker_v2_modem_module` application.
```

Comment:
```suggestion
  Hence, it is recommended to upgrade to the :ref:`asset_tracker_v2` application.
```

Reply:
Good catch.  I didn't write this message, just tried to fix the missing space after the first period.... I hadn't noticed this other error, thanks.